pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/Policies.java

```diff
@@ -18,15 +18,52 @@
  */
 package org.apache.pulsar.common.policies.data;
 
+import java.lang.reflect.Type;
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
 
 import com.google.common.base.MoreObjects;
 import com.google.common.collect.Lists;
 import com.google.common.collect.Maps;
+import com.google.common.reflect.TypeToken;
 
 public class Policies {
+    public enum PolicyProperty {
+        BacklogQuotaMap(new TypeToken<Map<BacklogQuota.BacklogQuotaType, BacklogQuota>>(){}.getType(), "backlog_quota_map", false),
```

Comment:
Why not reuse the same field name (`backlog_quota_map`, ..) even as REST handler, instead of using a different format like `BacklogQuotaMap`?